### PR TITLE
headplane_ynh

### PR DESCRIPTION
**Pull request by mdtita**

### CODE REVIEW

**Comment by ericgaspar** [comment]
you should clean up comments 🗑️

**Comment by mdtita** [comment]
> you should clean up comments 🗑️

done

**Comment by ericgaspar** [comment]
add also, place `Headplane` in the alphabetical list (after `headphones`)

**Comment by ericgaspar** [comment]
You are highly encouraged to move `headplane_ynh` to the [YunoHost-app](https://github.com/yunohost-apps) repo

**Comment by mdtita** [comment]
> You are highly encouraged to move `headplane_ynh` to the [YunoHost-app](https://github.com/yunohost-apps) repo

sorry, can you explain a bit further?

**Comment by ericgaspar** [comment]
you should be able to upload the app to the branch  https://github.com/YunoHost-Apps/headplane_ynh

**Comment by mdtita** [comment]
Done

File: apps.toml [hunk 1]
```diff
@@ -1663,6 +1663,11 @@ state = "working"
 subtags = [ "download" ]
 url = "https://github.com/YunoHost-Apps/headphones_ynh"
 
+[Headplane]
+category = "system_tools"
+state = "working"
+url = "https://github.com/mdtita/headplane_ynh"
```
Comment (tituspijean) [inline apps.toml:1669]:
```suggestion
url = "https://github.com/YunoHost-Apps/headplane_ynh"
```

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**